build/docker/proxy/assets/sbin/init.d/00-config.sh

```diff
@@ -4,3 +4,15 @@ set -e
 cp -fv /etc/nginx/nginx.conf.template /etc/nginx/nginx.conf
 
 sed -i "s;API_HOST_TO_REPLACE;${API_HOST//&/\\&};g" /etc/nginx/nginx.conf
+
+allowed_host_list=${API_ADMIN_ALLOWED_HOSTS}
+allowed_hosts=""
+
+IFS=","
+for allowed_host in $allowed_host_list
```

Comment:
Can we put `for` and `do` on the same line i.e. `for allowed_host in $allowed_host_list; do`?